logging/interceptor.go

```diff
@@ -39,6 +53,109 @@ func LogLevelInterceptor(defaultLevel logrus.Level) grpc.UnaryServerInterceptor
 	}
 }
 
+func ClientInterceptor(logger *logrus.Logger, opts ...Option) grpc.UnaryClientInterceptor {
+	o := evaluateClientOpt(opts)
+	return func(ctx context.Context, method string, req, reply interface{}, cc *grpc.ClientConn, invoker grpc.UnaryInvoker, opts ...grpc.CallOption) error {
+		startTime := time.Now()
+		fields := newLoggerFields(method, startTime, clientKind)
+		err := invoker(ctx, method, req, reply, cc, opts...)
+		addSpecificFields(ctx, fields, logger, o)
+		code := o.codeFunc(err)
+		level := o.levelFunc(code)
+		durField, durVal := o.durationFunc(time.Since(startTime))
+		fields["grpc.code"] = code.String()
+		fields[durField] = durVal
```

Comment:
This block has poor readability , it will be better to reduce LOC between initialization and actual usage of variable, as well please separe each field by blank line 

```go
startTime := time.Now()

fields := newLoggerFields(method, startTime, clientKind)
addSpecificFields(ctx, fields, logger, o)

err := invoker(ctx, method, req, reply, cc, opts...)
if err != nil {
 	fields[logrus.ErrorKey] = err
}

code := o.codeFunc(err)
fields[GRPCCodeKey] = code.String()

durField, durVal := o.durationFunc(time.Since(startTime))
fields[durField] = durVal

level := o.levelFunc(code)
levelLogf(logrus.NewEntry(logger).WithFields(fields), level, "finished unary call with code "+code.String())

return err
```